tests/integration/components/o-s-s/currency-input-test.ts

```diff
@@ -16,6 +16,44 @@ module('Integration | Component | o-s-s/currency-input', function (hooks) {
     assert.dom('.currency-input-container').exists();
   });
 
+  test('it renders currency only', async function (assert) {
```

Comment:
what about nesting all the tests you added into a `module('currency only mode')` ?